duckduckgo/duckduckgo-privacy-extension
# Privacy practices and Grade Scorecard

**Reviewer:** @laurengarcia 



## Description:


Adding the privacy practices view, still WIP.

![screenshot from 2017-12-27 15-10-40](https://user-images.githubusercontent.com/1431577/34385194-ee3f8820-eb18-11e7-9ba8-8835b59f0632.png)


## Steps to test this PR:

1. Remove the `is-hidden` on this line: https://github.com/duckduckgo/duckduckgo-privacy-extension/blob/7dbadaa5526ca9c0eaf41d81796149fba98109c1/shared/js/ui/templates/site.es6.js#L50
2. Visit a bunch of websites (e.g. steampowered.com, gmail.com, duckduckgo.com, etc) and open the privacy practices subview to see what this looks like.


## Automated tests:

###### Reviewer Checklist:


###### PR Author Checklist:
